build_modules/pubspec.yaml

```
@@ -9,7 +9,7 @@ environment:
 dependencies:
   analyzer: ">0.36.4 <0.42.0"
   async: ^2.0.0
-  bazel_worker: ^0.1.20
+  bazel_worker: ">=0.1.18 <2.0.0"
```

```suggestion
  bazel_worker: ">=0.1.20 <2.0.0"
```

Unless you need 0.1.18 to be selectable? Or we can try to confirm that 0.1.18 does work, I don't know if this was bumped unnecessarily or not.

---

oh woops - thanks.